modules/layers/src/base-tile-layer/utils/tile-cache.js

```diff
@@ -1,5 +1,33 @@
 import Tile from './tile';
 
+const TILE_STATE_UNKNOWN = 0;
+const TILE_STATE_VISIBLE = 1;
+/*
+   show cached parent tile if children are loading
+   +-----------+       +-----+            +-----+-----+
+   |           |       |     |            |     |     |
+   |           |       |     |            |     |     |
+   |           |  -->  +-----+-----+  ->  +-----+-----+
+   |           |             |     |      |     |     |
+   |           |             |     |      |     |     |
+   +-----------+             +-----+      +-----+-----+
+
+   show cached children tiles when parent is loading
```

Comment:
Two way reuse, both up and down... This is not used by 3d tiles. Does this complicate things much?

Reply:
3d-tiles loads everything from the root to the current level, so parent is always available. 2d only loads tiles at the current level. When you zoom out and no ancestors are available, the map will flash white and you lose all visual reference. Yes it adds complexity, but it's also needed for usability.